src/api/Feed.js

```diff
@@ -348,7 +355,7 @@ class Feed extends Base {
             : Promise.resolve();
         const versionsPromise = shouldShowVersions ? this.fetchVersions() : Promise.resolve();
         const currentVersionPromise = shouldShowVersions ? this.fetchCurrentVersion() : Promise.resolve();
-        const commentsPromise = this.fetchComments(permissions);
+        const commentsPromise = this.fetchComments(permissions, shouldShowReplies);
```

Comment:
Should we consider creating new methods rather than having the conditions in the inner method?

```
const commentsPromise = shouldShowReplies ? this.fetchThreadedComments : this.fetchComments;
```

Reply:
I've changed approach and, as suggested, created separate methods for all operations.